rpc.go

```diff
@@ -312,3 +310,153 @@ func getMiddlewareResult(u *User, line string) string {
 	}
 	return line
 }
+
+func pluginsCMD(_ string, u *User) {
+	plugins := make([]CMD, 0, len(PluginCMDs))
+	for n, c := range PluginCMDs {
+		plugins = append(plugins, CMD{
+			name:     n,
+			info:     c.info,
+			argsInfo: c.argsInfo,
+		})
+	}
+	autogenerated := autogenCommands(plugins)
+	if autogenerated == "" {
+		autogenerated = "   (no plugin commands are loaded)"
+	}
+	u.room.broadcast("", "Plugin commands  \n"+autogenerated)
+}
+
+type tokensDbEntry struct {
+	Token string `json:"token"`
+	Data  string `json:"data"`
+}
+
+var Tokens = make([]tokensDbEntry, 0)
+
+func initTokens() {
+	if Integrations.RPC == nil {
+		return
+	}
+
+	f, err := os.Open(Config.DataDir + string(os.PathSeparator) + "tokens.json")
+	if err != nil {
+		if !os.IsNotExist(err) {
+			Log.Fatal("Error reading tokens file: " + err.Error())
+		}
+		return
+	}
+	defer f.Close()
+	err = json.NewDecoder(f).Decode(&Tokens)
+	if err != nil {
+		error := "Error decoding tokens file: " + err.Error()
+		MainRoom.broadcast(Devbot, error)
+		Log.Fatal(error)
+	}
+}
+
+func saveTokens() {
+	f, err := os.Create(Config.DataDir + string(os.PathSeparator) + "tokens.json")
+	if err != nil {
+		Log.Fatal("Error saving tokens file: " + err.Error())
```

Comment:
```suggestion
		Log.Println("Error saving tokens file:", err)
```
Don't use `Log.Fatal` anywhere since it brings down the whole server.